test_data/upgrade/pr2094.trig

```diff
@@ -0,0 +1,12 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
+
+# bad valueHasUri value and missing datatype
+<http://rdfh.ch/0103/5LE8P57nROClWUxEPJhiug/values/fEbt5NzaSe6GnCqKoF4Nhg/standoff/2>
+    rdf:type                                knora-base:StandoffUriTag ;
+    knora-base:standoffTagHasEnd            50 ;
+    knora-base:standoffTagHasStart          20 ;
+    knora-base:standoffTagHasStartIndex     2 ;
+    knora-base:standoffTagHasStartParent     <http://rdfh.ch/0103/5LE8P57nROClWUxEPJhiug/values/fEbt5NzaSe6GnCqKoF4Nhg/standoff/1> ;
+    knora-base:standoffTagHasUUID           "dd2e785b-041e-4bcb-a77d-570efe8a068c" ;
+    knora-base:valueHasUri                  <http://www.maison-george-sand.fr/> .
```

Comment:
not sure if that makes much sense... but you could consider testing also the case where a URI should not be converted to a string?

Reply:
This plugin is very specific and picks only the node values in type `IriNode`. IMO as with the PR2079 plugin this is enough.